src/site/twitch.tv/modules/chat/components/ChatEmoteTooltip.vue

```diff
@@ -1,20 +1,75 @@
 <template>
 	<div ref="tooltip" class="seventv-tooltip" tooltip-type="emote">
-		<h3>{{ emote.name }}</h3>
+		<img ref="imgRef" class="tooltip-emote" :srcset="srcSet" :alt="emote.name" />
+		<div class="spacer">
+			<h3 class="emote-name">{{ emote.name }}</h3>
+			<div class="spacer"></div>
+			<Logo class="logo" :provider="emote.provider" />
+		</div>
 	</div>
 </template>
 
 <script setup lang="ts">
-defineProps<{
-	emote: SevenTV.Emote;
-}>();
+import { computed } from "vue";
+import Logo from "@/assets/svg/Logo.vue";
+
+const props = withDefaults(
+	defineProps<{
+		emote: SevenTV.ActiveEmote;
+		imageFormat?: SevenTV.ImageFormat;
+		unload?: boolean;
+	}>(),
+	{ unload: false, imageFormat: "WEBP" },
+);
+
+const srcSet = computed(() => (props.unload ? "" : getSrcSet(props.emote)));
+
+function getSrcSet(emote: SevenTV.ActiveEmote) {
+	const host = emote.data?.host ?? { url: "", files: [] };
+	const format = host.files.some((fi) => fi.format === props.imageFormat) ? props.imageFormat : host.files[0]?.format;
+
+	return host.files
+		.filter((f) => f.format === format)
+		.map((f, i) => `${host.url}/${f.name} ${i - 1}x`)
+		.join(", ");
+}
 </script>
 
 <style scoped lang="scss">
 .seventv-tooltip {
-	background-color: rgba(0, 0, 0, 0.65%);
-	backdrop-filter: blur(0.25em);
-	border-radius: 0.33em;
-	padding: 1em;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	min-width: 12em;
+	background-color: var(--seventv-background-transparent-2);
+	backdrop-filter: blur(16px);
+	outline: 1px solid var(--seventv-border-transparent-1);
+	border-radius: 0.25em;
+	padding: 1.3em;
+}
+.emote-name {
+	font-size: 16px;
+	font-weight: 300;
+	max-width: 300px;
+	word-break: break-all;
+	float: left;
+}
+
+.logo {
+	width: 24px;
+	height: auto;
+	float: right;
+	align-self: end;
+}
+.tooltip-emote {
+	padding-bottom: 3em;
+	height: auto;
+	max-width: 100px;
+}
+.spacer {
+	display: flex;
```

Comment:
Text and provider logo are too close. Perhaps add a gap to separate them

```suggestion
	display: flex;
	column-gap: 0.25em;
```

![image](https://user-images.githubusercontent.com/9599344/211385973-ae4dab99-904e-4432-8c2f-ad445cd027e7.png)
![image](https://user-images.githubusercontent.com/9599344/211386041-2e3c4763-3f5c-4c78-80a0-a7bda054b239.png)